xbmc/GUIInfoManager.cpp

```diff
@@ -6480,6 +6480,12 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///     @return The Musicbrainz release status of the album (offical, bootleg, promotion etc)
 ///     <p><hr>
 ///     @skinning_v19 **[New Infolabel]** \link ListItem_AlbumStatus `ListItem.AlbumStatus`\endlink
+///   \table_row3{   <b>`ListItem.Property(Album_Duration)`</b>,
+///                  \anchor ListItem_Property_Album_Duration
+///                  _string_,
+///     @return The duration of the album in HH:MM:SS.
+///     <p><hr>
+///     @skinning_v19 **[New Infolabel]** \link ListItem_Property_Album_Duration `ListItem.Property(Album_Duration)`\endlink
```

Comment:
Best keep properties together, move this to just below ListItem.Property(Album_Isboxset)

Reply:
Moved